### ISSUE TITLE
18c support

### ISSUE THREAD

**Issue by chrisdrew1** [biemond/biemond-oradb#235]
Is 18c support going to be added as the installer is image based now for the db.

**Comment by biemond** [comment]
Oh, I am planning to do this. I did not look at it yet. But I was thinking it is just a minor 12.2 release with a new 18c name . You have more info about this image part. 

Thanks

**Comment by chrisdrew1** [comment]
https://docs.oracle.com/en/database/oracle/oracle-database/18/ladbi/about-image-based-database-installation.html#GUID-0315AA15-1C03-4E96-B801-3A8E5F5D4B39

It's a MUCH faster install as you just unzip it into your oracle-home now.  Kind of like the newer grid installs.

**Comment by biemond** [comment]
Ok cool. Create an db is still the same with dbca optional with a template ?

**Comment by chrisdrew1** [comment]
Yep.  Still very much the same.

**Comment by biemond** [comment]
I guess I won’t support this golden image creation based on an existing oracle home. But nice Oracle is doing cloning and just extracting.

**Comment by biemond** [comment]
Did you try the rpm based installation?

**Comment by chrisdrew1** [comment]
So, no 18c support then?

**Comment by chrisdrew1** [comment]
I haven't tried the rpm based install yet.

**Comment by biemond** [comment]
Not yet , in a few weeks for sure. A lot of research and testing without breaking the current stuff

**Comment by chrisdrew1** [comment]
Great.  I'm still trying to find a good way to release PSU/RU to the environments without having puppet/opatch fail after every run.  opatch wants to apply patch, db is still up, puppet fails.

**Comment by biemond** [comment]
I need to take a look at puppet tasks for this .  If this can solve these tasks you need to do a few times a year

**Comment by chrisdrew1** [comment]
What we do now is deploy with the module, then manual patching afterwards..  Works, but it's not ideal.  An emcli fleet management module would be awesome.

**Comment by chrisdrew1** [comment]
I'm working using a hiera version value to have some machines not get patched, and new builds get the latest RU/PSU.

**Comment by finduramit** [comment]
Would be great to have 18c support soon. We have some resources lined up to test.
Thanks
Amit

**Comment by phince** [comment]
@biemond 
Is there an update on the roadmap for the implementation of 18c support?

**Comment by biemond** [comment]
yeah, there is. I know what to do , add it to the manifests and do a lot of testing.

I guess I will start with the client , database and at last grid.

**Comment by biemond** [comment]
On github oradb, 18.3 for grid and db install + config are working.  I made an example but there is an issue on github,  but basically use 18.0.0.0 as version for the install and 18.3 in the DB creation.  Use LINUX.X64_180000_db_home.zip or LINUX.X64_180000_grid_home.zip as zip.
